web/asgi.py

```
@@ -9,8 +9,21 @@
 
 import os
 
+import django
```

Pylint (3.3.7)

[error] Unable to import 'django'

(E0401)